src/components/LandingAnimation/LandingAnimation.js

```diff
@@ -88,15 +106,22 @@ const LandingAnimation = () => {
       >
         Press to say &apos;Hello!&apos;
       </StartButton>
-      <StyledSpan ref={spanHiRef}>Hi!</StyledSpan>
-      <StyledSpan ref={spanMyRef}>My</StyledSpan>
-      <StyledSpan ref={spanNameRef}>name</StyledSpan>
-      <StyledSpan ref={spanIsRef}>is</StyledSpan>
-      <StyledSpan ref={spanKarolRef}>Karol</StyledSpan>
+      <StyledSpan ref={hiSpanRef}>Hi!</StyledSpan>
```

Comment:
Will it be translated?

Reply:
yes, TODO comment placed and translation will be done in separate PR